aria-practices.html

```diff
@@ -710,100 +710,102 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
     </section>
 
     <section class="widget" id="combobox">
-      <h3>Combo Box</h3>
+      <h3>Combobox</h3>
       <p>
-        A <a href="#combobox" class="role-reference">combobox</a> is a widget made up of the combination of two distinct elements:
-        1) a single-line textbox, and 2) an associated pop-up element for helping users set the value of the textbox.
+        A <a href="#combobox" class="role-reference">combobox</a> is an input widget with an associated popup that enables users to select a value for the combobox from a collection of possible values.
+        In some implementations, the popup presents allowed values, while in other implementations, the popup presents suggested values , and users may either select one of the suggestions or type a value.
         The popup may be a <a href="#Listbox">listbox</a>,
         <a href="#grid">grid</a>,
         <a href="#TreeView">tree</a>, or
         <a href="#dialog_modal">dialog.</a>
-        Many implementations also include a third optional element -- a graphical button adjacent to the textbox, indicating the availability of the popup.
-        Activating the button displays the popup if suggestions are available.
+        Many implementations also include a third optional element -- a graphical <q>Open</q> button adjacent to the combobox, which indicates availability of the popup.
+        Activating the <q>Open</q> button displays the popup if suggestions are available.
       </p>
-      <p>The popup is hidden by default, and the conditions that trigger its display are specific to each implementation. Some possible popup display conditions include:</p>
+      <p>
+        The combobox pattern supports several optional behaviors.
+        The one that most shapes interaction is text input.
+        Some comboboxes allow users to type and edit text in the combobox and others do not.
+        If a combobox does not support text input, it is referred to as select-only, meaning the only way users can set a value is by selecting a value in the popup.
+        For example, in some browsers, an HTML <code>select</code> element with <code>size="1"</code> is presented to assistive technologies as a combobox.
+        Alternatively, if a combobox supports text input, it is referred to as editable.
+        An editable combobox may either allow users to input any arbitrary value, or it may restrict its value to a discrete set of allowed values, in which case typing input serves to filter suggestions presented in the popup.
+      </p>
+      <p>The popup is hidden by default, i.e., the default state is collapsed. The conditions that trigger expansion -- display of the popup --are specific to each implementation. Some possible conditions that trigger expansion include:</p>
```

Comment:
```suggestion
      <p>The popup is hidden by default, i.e., the default state is collapsed. The conditions that trigger expansion -- display of the popup -- are specific to each implementation. Some possible conditions that trigger expansion include:</p>
```
Added space after em dash following "display of the popup" to separate it from "are specifically."